tools/dm-query-status.md

```diff
@@ -0,0 +1,234 @@
+                        "recentTps": "1",       # The number of binlog events that are synchronized in this subtask in the last one second.
+                        "masterBinlog": "(bin.000001, 3234)",                               # The binlog position in the upstream database.
+                        "masterBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-14",    # The GTID information in the upstream database.
+                        "syncerBinlog": "(bin.000001, 2525)",                               # The position of the binlog that has been synchronized
+                                                                                            # in the `Sync` processing unit.
+                        "syncerBinlogGtid": "",                                             # It is always empty because `Sync` does not use GTID to
+                                                                                            # synchronize data.
+                        "blockingDDLs": [       # The DDL list that is blocked currently. It is not empty only when all the upstream tables of this 
+                                                # DM-worker are "synced". In this case, it indicates the sharding DDL statement to be executed or skipped.
+                            "USE `test`; ALTER TABLE `test`.`t_target` DROP COLUMN `age`;"
+                        ],
+                        "unresolvedGroups": [   # The sharding group that is not resolved.
+                            {
+                                "target": "`test`.`t_target`",                  # The downstream database table to be synchronized.
+                                "DDLs": [
+                                    "USE `test`; ALTER TABLE `test`.`t_target` DROP COLUMN `age`;"
+                                ],
+                                "firstPos": "(bin|000001.000001, 3130)",        # The starting position of the sharding DDL statement.
+                                "synced": [                                     # The sharded table of the sharding DDL statement that `Sync` has read.
+                                    "`test`.`t2`"
+                                    "`test`.`t3`"
+                                    "`test`.`t1`"
+                                ],
+                                "unsynced": [                                   # The upstream table of the sharding DDL statement that has not been
+                                                                                # executed. If any upstream tables have not been synchronized completely,  
+                                                                                # `blockingDDLs` is empty.
+                                ]
+                            }
+                        ],
+                        "synced": false         # Whether the progress of incremental synchronization in the downstream has been the same as that in the 
```

Comment:
In fact there is no `incremental synchronization` in upstream. what about ``Whether the incremental synchronization catches up with upstream and has the same binlog position as that in the upstream``